carbon-devnet-0/carbon-devnet-0-scripts/setup.sh

```diff
@@ -50,6 +50,40 @@ if [ ! -f "/usr/local/lib/libleveldb.so.1" ]; then
   sudo service postgresql restart
 fi
 
+if [ ! -f "/usr/local/lib/librocksdb.so.7" ]; then
```

Comment:
delete setup.sh, add peers and version file

Reply:
removed /carbon-devnet-0/ directory